src/test/java/de/eposcat/master/generators/CommonCaseSetup.java

```diff
@@ -94,6 +89,9 @@ public void simplePerformanceTest() {
                     log.info("Warm-up is done. Lets do this!!!");
                 }
                 testChanges(ADAPTERS_MAP.get(approachName), approachName);
+                testAttributeNameQuery(ADAPTERS_MAP.get(approachName), approachName);
+                testAttributeValue(ADAPTERS_MAP.get(approachName), approachName, new AttributeBuilder().setValue("true").setType(AttributeType.String).createAttribute());
+                changeSingleBigAttributeTest(ADAPTERS_MAP.get(approachName), approachName);
```

Comment:
IMHO these methods should be called from separate @Test methods for 2 reasons:
- we want to test each test case independently
- if the test for e.g. testAttributeValue fails, this should not prevent us from getting the results from the other use cases.